lib/pstore.rb

```diff
@@ -147,27 +345,43 @@ def in_transaction_wr
   end
   private :in_transaction, :in_transaction_wr
 
+  # :call-seq:
+  #   pstore[key]
```

Comment:
We don't need a call-seq for these since they're in Ruby so RDoc is able to determine their signature.

Reply:
I added five call-seq directives to override the argument name `name` with `key`.  I think the distinction is important.  An alternative would be to make the change in code, which I personally would not want to do.